crates/fluvio-cli/src/error.rs

```diff
@@ -28,6 +28,8 @@ pub enum CliError {
     OutputError(#[from] OutputError),
     #[error("Failed to parse format string")]
     TemplateError(#[from] TemplateError),
+    #[error("Failed to generate new SmartModule")]
+    GenerateError(String),
```

Comment:
The error type used in `cargo-generate` is not exported. Checking on the source code I saw they are using `anyhow` but is not exported, so I'm taking the error message as a `String` and passing it to the bubbled error instance.

Reply:
Let's use anyhow everywhere.

Reply:
Yeah! I noticed smdk already uses it.
This code doesn't lives here anymore because it was moved to `smdk` crate.